src/lib/components/utils.js

```diff
@@ -1,3 +1,7 @@
 export const prettyPrintBooleanValue = (value) => {
   return value ? 'Yes' : 'No';
 };
+
+export const screenIsWiderThan = (pixeles) => {
```

Comment:
Good solution in the end!

Very minor typo, but you can keep it as it is or fix in another PR!
```suggestion
export const screenIsWiderThan = (pixels) => {
```